### PR TITLE
Settings page

### PR DESCRIPTION
**Pull request by Haren-Venket**
Done frontend. backend upload work left and setting information on start up

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -7,7 +7,10 @@
     "@emotion/styled": "^11.8.1",
     "@fontsource/roboto": "^4.5.5",
     "@mui/icons-material": "^5.5.1",
+    "@mui/styles": "5.5.3",
     "@mui/material": "^5.5.3",
+    "@mui/core": "5.0.0-alpha.54",
+    "classname": "0.0.0",
```
Comment (shad0whe1ix) [inline client/package.json]: remove this dependency

File: client/src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <Link to="/student">Student</Link> |{" "}
+        <Link to="/organization">Organization</Link>
+        <Link to="/settings">Settings</Link>
+        <Link to="/user/:id">user page</Link>
+      </nav>
+    </div>
+  );
+}
+
+export default App;
```
Comment (shad0whe1ix) [inline client/src/components/App.js]: delete this file, we use `App.jsx` now

File: client/src/index.js [hunk 3]
```diff
@@ -26,6 +29,8 @@ render(
       <Route path="/" element={<App />}></Route>
       <Route path="/signin" element={<SignIn />}></Route>
       <Route path="/signup" element={<SignUp />}></Route>
+      <Route path="/student" element={<Student />}></Route>
```
Comment (shad0whe1ix) [inline client/src/index.js]: remove this, I deleted the student page because its unnecessary

**Review by shad0whe1ix — CHANGES_REQUESTED** [review]

File: client/package.json (hunk 1 above)
Comment (Haren-VenketS) [reply client/package.json]: already deleted

File: client/src/components/App.js (hunk 2 above)
Comment (Haren-VenketS) [reply client/src/components/App.js]: done

File: client/src/index.js (hunk 3 above)
Comment (Haren-VenketS) [reply client/src/index.js]: Deleted and Resolved

**Review by shad0whe1ix — APPROVED** [review]

**Pull request merged**